### PR TITLE
Added Contributing.md file

### PR DESCRIPTION
**Pull request by aysp-99**
Addresses #8 

- Added `CONTRIBUTING.md` file with relevant information mentioned in the issue description.
- Created a `utils` folder which contains relevant screenshots used in ` CONTRIBUTING.md` file.

### CODE REVIEW

**Review by Siddhant-K-code — APPROVED** [review]
LGTM

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+* [Learn Git: Everything You Need To Know](https://www.udemy.com/course/learngit/?ranMID=39197&ranEAID=JVFxdTr9V80&ranSiteID=JVFxdTr9V80-K5pDC6IWSDnHLwh8jTAbEg&LSNPUBID=JVFxdTr9V80&utm_source=aff-campaign&utm_medium=udemyads)
+* [The beginner's guide to contribute to a GitHub project](https://akrabat.com/the-beginners-guide-to-contributing-to-a-github-project/)
+
+
+
+
+
+
+
+
```
Comment (HarshCasper) [inline CONTRIBUTING.md]: Why are there so many extra lines?

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Please remove all the Images from the `utils` folder. Instead upload these images on Imgur and reference it from there. 

Thanks!

File: CONTRIBUTING.md (hunk 1 above)
Comment (aysp-99) [reply CONTRIBUTING.md]: Sorry, I didn't notice it. 
Will fix it.

**Comment by aysp-99** [comment]
> Please remove all the Images from the `utils` folder. Instead upload these images on Imgur and reference it from there.
> 
> Thanks!

Done. 👍

**Review by Siddhant-K-code — CHANGES_REQUESTED** [review]
image
should we just remove extra spaces from the lines of commands to ``` commands ``` from 
```
commands
```
it will look better and cleaner. 

what're your thoughts @HarshCasper ?

**Comment by HarshCasper** [comment]
> image
> should we just remove extra spaces from the lines of commands to `commands` from
> 
> ```
> commands
> ```
> 
> it will look better and cleaner.
> 
> what're your thoughts @HarshCasper ?

Doesn't make much difference. Would it be great if @aysp-99 can make it happen. 

Also, since everything looks fine, can you push all your changes into a single commit, so that I can merge it? If you don't know how to do that, I can help you 👍

**Comment by aysp-99** [comment]
> > image
> > should we just remove extra spaces from the lines of commands to `commands` from
> > ```
> > commands
> > ```
> > 
> > 
> > it will look better and cleaner.
> > what're your thoughts @HarshCasper ?
> 
> Doesn't make much difference. Would it be great if @aysp-99 can make it happen.
> 
> Also, since everything looks fine, can you push all your changes into a single commit, so that I can merge it? If you don't know how to do that, I can help you 👍

I made the necessary changes to it.
I tried to squash the previous commits but I am not sure if it worked. I am not quite well versed with 'squash and rebase workflow'. Can you please guide me?

**Review by HarshCasper — APPROVED** [review]

**Comment by HarshCasper** [comment]
> > > image
> > > should we just remove extra spaces from the lines of commands to `commands` from
> > > ```
> > > commands
> > > ```
> > > 
> > > 
> > > it will look better and cleaner.
> > > what're your thoughts @HarshCasper ?
> > 
> > 
> > Doesn't make much difference. Would it be great if @aysp-99 can make it happen.
> > Also, since everything looks fine, can you push all your changes into a single commit, so that I can merge it? If you don't know how to do that, I can help you 👍
> 
> I made the necessary changes to it.
> I tried to squash the previous commits but I am not sure if it worked. I am not quite well versed with 'squash and rebase workflow'. Can you please guide me?

Never mind. I have merged it 👍 

Let me explain how you can do this in future: 

So, lets say: there were 5 commits in total:

`git reset --soft HEAD~5` will soft reset the last 5 commits.

Then, you can make a single commit by doing the standard `git add` and `git commit`.

And after that, `git push -f origin HEAD` will force push the single commit.

**Comment by aysp-99** [comment]
> > > > image
> > > > should we just remove extra spaces from the lines of commands to `commands` from
> > > > ```
> > > > commands
> > > > ```
> > > > 
> > > > 
> > > > it will look better and cleaner.
> > > > what're your thoughts @HarshCasper ?
> > > 
> > > 
> > > Doesn't make much difference. Would it be great if @aysp-99 can make it happen.
> > > Also, since everything looks fine, can you push all your changes into a single commit, so that I can merge it? If you don't know how to do that, I can help you 👍
> > 
> > 
> > I made the necessary changes to it.
> > I tried to squash the previous commits but I am not sure if it worked. I am not quite well versed with 'squash and rebase workflow'. Can you please guide me?
> 
> Never mind. I have merged it 👍
> 
> Let me explain how you can do this in future:
> 
> So, lets say: there were 5 commits in total:
> 
> `git reset --soft HEAD~5` will soft reset the last 5 commits.
> 
> Then, you can make a single commit by doing the standard `git add` and `git commit`.
> 
> And after that, `git push -f origin HEAD` will force push the single commit.

Oh, got it.
Thanks. 👍

**Pull request merged**